### ISSUE TITLE
Creating your own dataset 

### ISSUE THREAD

**Issue by Yaffa1607** [Hvass-Labs/TensorFlow-Tutorials#3]
Hi , I am trying to train the model on my own dataset. 
In what format should I  prepare my dataset?

**Comment by Hvass-Labs** [comment]
This is described in Tutorial #09. Please watch the video and read the comments in the source-code, which describes the format.

**Comment by Yaffa1607** [comment]
Hi , I tried running the Tutorial 9 code for my own dataset. 
It did not create a test set for any of the image folders. 
I have a folder with multiple hand gestures that I want to use to train the VGG16 image segmentation  to detect hands in images.  
This is the network I am using : 
https://github.com/MarvinTeichmann/tensorflow-fcn/blob/master/fcn16_vgg.py

**Comment by Hvass-Labs** [comment]
The dir-structure is described here:

https://github.com/Hvass-Labs/TensorFlow-Tutorials/blob/master/dataset.py#L59-L92

It is also discussed in the video for Tutorial #09 (I get the impression that you haven't seen it?)

https://www.youtube.com/watch?v=9sYBC2_AVfA

I'm afraid I can't help you with the VGG16 model.

**Comment by Yaffa1607** [comment]
Hey , yeah figured it out. I had errors because my labels were binary
images.
Thanks again ,
Looking forward to your tutorials.

Best ,
Yasha

**Comment by samadak** [comment]
Hi dear DR Pedersen 

I am new in tensorflow and i wanna to creat my dataset(Logo32 filickr)4240 photos.I have 32 classes and 32 folders.I was wondering if you could help me to creat lable for my dataset. I mean a txt file for label that include like this:
img1 1(class1)
img2  2
..
Kind regard

**Comment by rdhiman56** [comment]
Hi, I am new to Tensorflow : installation is done it also giving sample hardcode reply but i want intigrate this with botkit can anybody know the flow how we can do that ????

Thanks in Advance!

**Comment by divya1md** [comment]
Hello, I am new to Tensorflow. I would like to train an image dataset for image recognition in Tensorflow. I could nt get how to train an image ourself. Please respond to this question as soon as possible because many were asking the same question

**Comment by usu008** [comment]
i developed the custom object detector with some 96 images (train 76 and test 20) after trained 200k steps my losses(1) goes down correctly....after that i create the interference graph all ok..........but my question is when i run my code to detect the object from video it not able to find the correct object ........ thanks in advance.....

actually i think we have a enough image for the dataset...because my dataset can able to detect the object from pre built video....so....it can detect the object from webcam also but not ...i dont kw the exact problem.....i think the resolution may b differ...thats y it not able to detect the object....﻿

use predefined "object detection" code with my own data set
ubuntu 16.04
tensorflow installed in anaconda environment
tensorflow 1.4.0
bazel 0.4.5
cpu version

**Comment by tjmac** [comment]
@Hvass-Labs I have watched the videos and would still like to learn how to label my own data set properly. Whether it be in a one hot encoding my own labels for my input image data set? Thank you!

**Comment by adjagetia** [comment]
Manually tagging the data is not more a hurdle. Check this out,
https://www.youtube.com/watch?v=tYqnsp-OcLQ
https://www.youtube.com/watch?v=xkSEnDIlvhI
you can quickly create your own image and video segmentation data in no time!!

Neurala’s Brain Builder Tool: https://hubs.ly/H0dkvRK0
